src/main/java/pl/sda/travelagency/AddTripDTO.java

```diff
@@ -0,0 +1,39 @@
+package pl.sda.travelagency;
+
+import lombok.*;
+
+import javax.validation.constraints.Digits;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+import java.math.BigDecimal;
+
+
+@Getter
+@Setter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class AddTripDTO {
+   @NotNull(message = "can't be empty!")
+    private Long destinationCityId;
```

Comment:
czy administrator dodając wycieczkę wybiera miasto po ID? To nie powinno być przypadkiem stringiem?

Reply:
`<option th:each="city : ${cities}" th:value="${city.getId()}" th:text="${city.getName()}"></option>` wybiera miasto po jego nazwie, a dalej jest przekazywane id i na jego podstawie jest później pobierane miasto z bazy. Eliminuje to potrzebę pisania metody wyszukiwania po nazwie.